records_mover/url/resolver.py

```diff
@@ -1,14 +1,62 @@
+from urllib.parse import urlparse
 from .base import BaseFileUrl, BaseDirectoryUrl
-from . import FileUrl, DirectoryUrl
-from typing import Any
+# TODO: Should I move these data structures here?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker